lib/ansible/plugins/become/pbrun.py

```diff
@@ -14,6 +14,7 @@
     options:
         become_user:
             description: User you 'become' to execute the task
+            default: root
```

Comment:
no, `default: ''` as there are cases you don't want to specify the user, even though the tool defaults to root, this would force root always and not the 'tool default'